Assets/Scripts/PlayerFlash.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+public class PlayerFlash : MonoBehaviour
+{
+    [SerializeField]
+    private Camera playerView;
+    [SerializeField]
+    private GameObject enemyObject;
```

Comment:
I think the enemy should be retrieved using the physics system, raycast, or sphere cast. In this configuration, the enemy object needs to be assigned at all times to prevent error messages. This is not guaranteed in all scenes.